server.js

```diff
@@ -1,9 +1,17 @@
 const express=require("express");
+const mongoose= require("mongoose");
 const app=express();
 const port=3000
 
+const data='mongodb+srv://aniketg:FOT8yjNAFZj39q0m@cluster0.wikddym.mongodb.net/?retryWrites=true&w=majority'
+
 app.get("/",(req,res)=>{
-    res.send("Weirdy Foody");
+    mongoose.connect(data)

```

Comment:
Consider moving the database connection to a separate function or module. This will make the code more modular and easier to maintain. [important]